js/markdown/shared/MarkdownCode.svelte

```diff
@@ -73,15 +73,18 @@
 			}
 		}
 	}
-	afterUpdate(() => render_html(message));
+
+	afterUpdate(async () => {
+		if (el && document.body.contains(el)) {
+			await render_html(message);
+		} else {
+			console.error("Element is not in the DOM");
+		}
+	});
 </script>
 
 <span class:chatbot bind:this={el} class="md" class:prose={render_markdown}>
-	{#if render_markdown}
-		{@html html}
-	{:else}
-		{html}
-	{/if}
```

Comment:
do we need this conditional? Removing this fixes the error. And we already handle the `render_narkdown` check in `process_message`

Reply:
its actually wrong anyway i think even if we did want to do it. Safe to remove.